docs/upload-image.md

```diff
@@ -4,6 +4,8 @@ To upload virtual machine images in the **Images** page, enter a URL that can be
 
 Currently, we support qcow2, raw, and ISO images.
 
+> Note: uploading images from the local desktop to the Harvester cluster is not supported yet, will be tracked on [#570](https://github.com/harvester/harvester/issues/570).
```

Comment:
Should we change the term `upload` in this file?

Reply:
probably leave it as what it is since this is a not-supported feature.